### DISCUSSION TITLE
Clarity Design module was not working in Angular 17.

### DISCUSSION THREAD

**Discussion by suryakumar-exe** [vmware-clarity/ng-clarity#1350] [Q&A] [answered]
After I Implemented Clarity design DataGrid, Its showing like below
![image](https://github.com/vmware-clarity/ng-clarity/assets/66153984/4d358ae8-dfc6-45dc-90db-a2f83a936c1d)

**Comment by kevinbuhmann** [comment, marked as answer]
Please make sure you have the styles imported properly..https://clarity.design/pages/developing
